test/dotnet-build.Tests/GivenDotnetBuildBuildsDcproj.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.Cli.Build.Tests
+{
+    public class GivenDotnetBuildBuildsDcproj : TestBase
+    {
+        [Fact]
+        public void ItPrintsBuildSummary()
+        {
+            var testAppName = "docker-compose";
+            var testInstance = TestAssets.Get(testAppName)
+                .CreateInstance(testAppName)
+                .WithSourceFiles()
+                .WithRestoreFiles();
+
+            string expectedBuildSummary = @"Build succeeded.
+    0 Warning(s)
+    0 Error(s)";
+
+            var cmd = new BuildCommand()
+                .WithWorkingDirectory(testInstance.Root)
+                .ExecuteWithCapturedOutput();
+            cmd.Should().Pass();
+            cmd.StdOut.Should().ContainVisuallySameFragment(expectedBuildSummary);
+        }
```

Comment:
This change is to only add Sdk.props and Sdk.targets of Microsoft.Docker.Sdk to CLI so that build won't fail when user builds a solution that includes both .NET Core projects and Docker Compose projects in command line. The Sdk.props and Sdk.targets only have the necessary targets to make the build of .dcproj a no-op. So there is no output at all.